packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -0,0 +1,372 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
```

Comment:
I don't have an experience with flow so, just learning here. Any tips for good tutorial?